frontend_tests/node_tests/user_events.js

```diff
@@ -23,9 +23,25 @@ set_global('pm_list', {
     update_private_messages: function () {},
 });
 
+set_global('narrow_state', {
+    update_email: function () {},
+});
+
+set_global('compose', {
+    update_email: function () {},
+});
+
+set_global('settings_account', {
+    update_email: function () {},
+});
+
 set_global('message_live_update', {
 });
 
+set_global('blueslip', {
+    error: function () {},
```

Comment:
Yep, I agree with @lonerz here, you can defer defining it until it actually needs to be stubbed.